core/templates/dev/head/pages/exploration_editor/history_tab/history_tab.html

```diff
@@ -31,6 +31,10 @@ <h3>List of Changes</h3>
         <[versionCountPrompt]>
       </div>
     </div>
+
+    <br>
```

Comment:
Better to add a style (margin-top?) on the div below rather than to use a br tag for whitespace.

Reply:
Done.